packages/ra-ui-materialui/src/form/SimpleFormConfigurable.tsx

```diff
@@ -0,0 +1,128 @@
+import * as React from 'react';
+import {
+    useResourceContext,
+    usePreference,
+    useStore,
+    useTranslate,
+} from 'ra-core';
+
+import { Configurable } from '../preferences';
+import { SimpleForm, SimpleFormProps } from './SimpleForm';
+import { SimpleFormEditor } from './SimpleFormEditor';
+
+export const SimpleFormConfigurable = ({
+    preferenceKey,
+    omit,
+    ...props
+}: SimpleFormConfigurableProps) => {
+    const translate = useTranslate();
+    const resource = useResourceContext(props);
+    const finalPreferenceKey = preferenceKey || `${resource}.simpleForm`;
+
+    const [availableInputs, setAvailableInputs] = useStore<
+        SimpleFormConfigurableColumn[]
+    >(`preferences.${finalPreferenceKey}.availableInputs`, []);
+
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    const [_, setOmit] = useStore<string[]>(
+        `preferences.${finalPreferenceKey}.omit`,
+        omit
+    );
+
+    React.useEffect(() => {
+        // first render, or the preference have been cleared
+        const inputs = React.Children.map(props.children, (child, index) =>
+            React.isValidElement(child)
+                ? {
+                      index: String(index),
+                      source: child.props.source,
+                      label:
+                          child.props.source || child.props.label
+                              ? child.props.label
+                              : translate(
+                                    'ra.configurable.simpleForm.unlabeled',
+                                    {
+                                        input: index,
+                                        _: `Unlabeled input #%{input}`,
+                                    }
+                                ),
+                  }
+                : null
+        ).filter(column => column != null);
+        if (JSON.stringify(inputs) !== JSON.stringify(availableInputs)) {
+            setAvailableInputs(inputs);
+            setOmit(omit);
+        }
+    }, [availableInputs]); // eslint-disable-line react-hooks/exhaustive-deps
+
+    return (
+        <Configurable
+            editor={<SimpleFormEditor />}
+            preferenceKey={finalPreferenceKey}
+            sx={{
+                display: 'block',
+                '&.RaConfigurable-editMode': {
+                    margin: '2px',
```

Comment:
MAde necessary because the outline is invisible for elements used as children of Paper, which has overflow:hidden